src/modules/Dropdown/Dropdown.js

```diff
@@ -385,10 +385,10 @@ export default class Dropdown extends Component {
 
   selectHighlightedItem = (e) => {
     const { multiple, onAddItem, options } = this.props
-    const value = _.get(this.getSelectedItem(), 'value')
+    const { value, disabled } = this.getSelectedItem() || {}
```

Comment:
If the first item is disabled, it will be "selected" by default since the selectedIndex is set to 0. 

One option to fix this would be to have a `getFirstValidItem` function or something (open to a better name) that would give the index of the first non-disabled item. If none (e.g. all options disabled) the selected index could be `-1`.